packages/guardian-prover-health-check-ui/static/config/hekla/guardians.json

```diff
@@ -11,5 +11,6 @@
 	"0x0EFADbE8328a1a07bB3dDEC22F69F88aDB61B8b4": "Gooner",
 	"0x55792e1F0a41D3af8B6d41DFdcf24651AA80fA1e": "Swift",
 	"0xD32b63e3bc54403d5363525b99E51E628c7c195e": "Casper",
-	"0xBba27Bd59eFe4F169afBa7b80a641a4675AD1Ff7": "Crash"
```

Comment:
And lets change `Lock`'s address to `0xaeBb6E8A857C86Fc45dBc51551b8E8B7a5791ab3`, this is the address he is using rn.

Reply:
fixed